pkg/resource/podtemplatespec.go

```diff
@@ -229,7 +229,24 @@ func makePodTemplateSpec(coreClient coreset.CoreV1Interface, proxyLister configl
 	}
 	volumes = append(volumes, vol)
 	mounts = append(mounts, corev1.VolumeMount{Name: vol.Name, MountPath: "/etc/pki/ca-trust/source/anchors"})
-	deps.AddConfigMap(vol.VolumeSource.ConfigMap.LocalObjectReference.Name)
+	deps.AddConfigMap(v1.ImageRegistryCertificatesName)
+
+	// Cluster trusted certificate authorities - mount to /usr/share/pki/ca-trust-source
+	// Registry runs update-ca-trust extract on startup, which merges the registry CAs with the cluster's trusted CAs
+	// into a single CA bundle.
+	vol = corev1.Volume{
+		Name: "trusted-ca",
```

Comment:
nit: I think you could use params.TrustedCA.Name here also, but this might be cleaner.

Reply:
will do once the ca controller lands